plutus-metatheory/src/Main.lagda

```diff
@@ -557,8 +557,9 @@ runU : TermU → Either ERROR TermU
 runU t = do
```

Comment:
Now this `run` function has a different behaviour to all the other `run` functions in this module, right? I'd probably have called it something else, but :shrug:

Reply:
I will add a comment but I expect we'll change the TCEK too when I'm there so it's just in the mean time.